helpdesk-ai/src/main/java/org/symphonyoss/symphony/bots/ai/conversation/IdleTimerManager.java

```diff
@@ -0,0 +1,61 @@
+package org.symphonyoss.symphony.bots.ai.conversation;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Class to manage Idles
```

Comment:
Please improve the component description.

Reply:
ok